pkg/api/validation/validation.go

```diff
@@ -1087,6 +1087,12 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 	if numVolumes == 0 {
 		allErrs = append(allErrs, field.Required(specPath, "must specify a volume type"))
 	}
+
+	// do not allow hostPath mounts of '/' to have a 'recycle' reclaim policy
+	if pv.Spec.HostPath != nil && pv.Spec.HostPath.Path == "/" && pv.Spec.PersistentVolumeReclaimPolicy == api.PersistentVolumeReclaimRecycle {
```

Comment:
+1 to `path.Clean`

> Why is HostPath recycle policy allowed anyway?

Personally, I don't think it should be, but it's V1 API now...